EIPS/eip-666.md

```diff
@@ -0,0 +1,47 @@
+---
+eip: 666
```

Comment:
```suggestion
eip: 4887
```
Filename should also be changed to `EIPS/eip-4887.md`

Reply:
Take it up with St. John ;)